scripts/linters/html_linter.py

```diff
@@ -38,6 +39,10 @@ class TagMismatchException(Exception):
 class CustomHTMLParser(html.parser.HTMLParser):
     """Custom HTML parser to check indentation."""
 
+    _SPACE_AROUND_ATTRIBUTE_REGEX = re.compile(
+        r' += +["{]| +=["{]|[^=]= +["{]'
```

Comment:
@DubeySandeep the regex is 3 or conditions:

1. ` += +["{]` one or more white spaces followed by a `=` followed again by one or more white spaces which are then followed by either " or {. eg: `  =  {`, `  =  "`. This is a condition where there are spaces around both left and right of `=`
2. ` +=["{]` one or more white spaces followed by `=` which is then followed by either " or {. This is when there is space in only the left-hand side. eg: `  ={`, `   ="`
3. `[^=]= +["{]` any character except `=` followed by `=` which is then followed by one or more white space, followed by either " and {. This is a condition where there are white spaces to the right of `=`. eg: `=  {`, `=  "`.

The usage of " and { is because all attribute values are either enclosed in double quotes or are commands which are enclosed in curly brackets.